llvm/lib/CodeGen/SelectionDAG/SelectionDAGBuilder.cpp

```diff
@@ -9130,6 +9142,15 @@ void SelectionDAGBuilder::visitCall(const CallInst &I) {
         if (visitBinaryFloatCall(I, ISD::FMAXNUM))
           return;
         break;
+      case LibFunc_fminimum_num:
+      case LibFunc_fminimum_numf:
+        if (visitBinaryFloatCall(I, ISD::FMINIMUMNUM))
+          return;
+        break;
+      case LibFunc_fmaximum_num:
```

Comment:
Methinks there's quite a few missing case statements here.

Reply:
LibFunc_fminimum_numl
LibFunc_fmaximum_numf
LibFunc_fmaximum_numl

Add. Should we add something like `fmaximum_numfN` and `fmaximum_numfNx `?

Reply:
If need, I think that we need to a new patch, since neither other  libcalls have them.